platform/cog-platform-drm.c

```diff
@@ -203,12 +203,24 @@ init_drm (void)
 
     const char* user_selected_mode = g_getenv("COG_PLATFORM_DRM_VIDEO_MODE");
 
+    const char* user_max_width_str = g_getenv("COG_PLATFORM_DRM_VIDEO_MAX_WIDTH");
+    const char* user_max_height_str = g_getenv("COG_PLATFORM_DRM_VIDEO_MAX_HEIGHT");
```

Comment:
Could we use a single environment variable for this? Something like `COG_PLATFORM_DRM_MODE_MAX=1920x1080` would be nicer, instead of having to define two different variables for width and height. If one only wants to limit height, a value of `0x1080` could be used, or `1920x0` to limit only the width.

Reply:
yeah I think that's also ok

Reply:
@aperezdc I made the changes, you can check it out